features/step_definitions/sample_file_steps.rb

```diff
@@ -135,7 +135,7 @@ def initialize
 end
 
 Given(/^a masking configuration file in the HOME directory$/) do
-  set_env('HOME', File.expand_path(File.join(current_directory, 'home')))
+  set_env 'HOME', Pathname("#{__dir__}/../../#{current_directory}/home").to_s
```

Comment:
I'm a little confused by this change. File.expand_path uses the working directory, which is conveniently set by Aruba.  `__dir__`, however, refers to the current file's directory. This means this will now break if Aruba moves its temporary directories elsewhere, or if we move the step definitions.

Also, why not use Pathname.join? Or `Pathname(File.expand_path ... )`?